Sparkle/SUSystemProfiler.m

```diff
@@ -36,7 +37,7 @@ @implementation SUSystemProfiler
     return [[NSDictionary alloc] initWithContentsOfFile:path];
 }
 
-+ (NSArray<NSDictionary<NSString *, NSString *> *> *)systemProfileArrayForHost:(SUHost *)host
++ (NSArray<NSDictionary<NSString *, id> *> *)systemProfileArrayForHost:(SUHost *)host
```

Comment:
If properties aren't guaranteed to be strings, you have to document what types they have. But I'd prefer to keep them all as strings for simplicity. They're going to be sent in a URL anyway.

Reply:
I was a bit hesitant to change it people were assuming certain values might be NSNumbers, but I suppose this is the first time this has technically be exposed publicly so that should be fine. 

I'll get all the values converted to strings tomorrow and revert all the type declarations.

Reply:
I've updated everything to strings but I've had to make some slight tweaks (they shouldn't really affect much):

1. For some reason we were dividing the CPU frequency by 1000 for the display frequency so that it would show in GHz. Given the CPU frequency and the readable label explicitly state MHz I removed that change
2. The RAM was being added as a float. In reality it seems it was always been treated as an int so it shouldn't affect anyone (unless someone desperately needs to know if someone has someone installed a fraction of a MB of RAM)
3. I've removed calls to `-description` when parameterising the feed URL as these are liable to break and are now unnecessary as everything is a string